db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(primary_key=True, max_length=255)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(primary_key=True, max_length=255)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.PROTECT)
+
+
+class Guild(models.Model):
+    name = models.CharField(primary_key=True, max_length=255)
```

Comment:
also unique

Reply:
btw, `primary_key` as a string value is not a best practice. So you shouldn't change it